contracts/validator-manager/MessageSpec.md

```diff
@@ -1,185 +1,155 @@
 
-- Description: Confirms conversion to a Permissionless Subnet on the P-Chain
+Description: Confirms conversion to L1 on the P-Chain
 
-- Signed by: P-Chain
-
-- Consumed by: Validator Manager contract
+Signed by: P-Chain
+Consumed by: L1
 
 Specification:
 
-```
-SubnetConversionMessage
-+--------------------+----------+----------+
-|            codecID :   uint16 |  2 bytes |
-+--------------------+----------+----------+
-|             typeID :   uint32 |  4 bytes |
-+--------------------+----------+----------+
-| subnetConversionID : [32]byte | 32 bytes |
-+--------------------+----------+----------+
-                                | 38 bytes |
-                                +----------+
-```
-
-where `subnetConversionID` is defined as the `sha256` hash of the serialization of the `SubnetConversionData`, defined as follows:
-
-```
-ValidatorData
-+--------------+----------+------------------------+
-|       nodeID :   []byte |  4 + len(nodeID) bytes |
-+--------------+----------+------------------------+
-| blsPublicKey : [48]byte |               48 bytes |
-+--------------+----------+------------------------+
-|       weight :   uint64 |                8 bytes |
-+--------------+----------+------------------------+
-                          | 60 + len(nodeID) bytes |
-                          +------------------------+
-
-SubnetConversionData
-+----------------+-----------------+--------------------------------------------------------+
-|       codecID  :          uint16 |                                                2 bytes |
-+----------------+-----------------+--------------------------------------------------------+
-|       subnetID :        [32]byte |                                               32 bytes |
-+----------------+-----------------+--------------------------------------------------------+
-| managerChainID :        [32]byte |                                               32 bytes |
-+----------------+-----------------+--------------------------------------------------------+
-| managerAddress :          []byte |                          4 + len(managerAddress) bytes |
-+----------------+-----------------+--------------------------------------------------------+
-|     validators : []ValidatorData |                        4 + sum(validatorLengths) bytes |
-+----------------+-----------------+--------------------------------------------------------+
-                                   | 74 + len(managerAddress) + len(validatorLengths) bytes |
-                                   +--------------------------------------------------------+                          
-```
-
-### `RegisterSubnetValidatorMessage`
-
-- Description: Registers a Subnet Validator on the P-Chain
-
-- Signed by: Subnet
-
-- Consumed by: P-Chain
-
-Specification:
+The `SubnetToL1ConversionMessage` is specified as an `AddressedCall` with `sourceChainID` set to the P-Chain ID, the `sourceAddress` set to an empty byte array, and a payload of:
```

Comment:
This line is a bit odd formatting wise. We also don't mention the `AddressedCall` format anywhere else. My preference would be to remove it.